js/src/util.js

```diff
@@ -116,7 +116,7 @@ const Util = (($) => {
     },
 
     reflow(element) {
-      new Function('bs', 'return bs')(element.offsetHeight)
+      return element.offsetHeight
```

Comment:
`javascript` write without `_` (undescore) https://houndci.com/configuration#javascript.

I think `java_script` is ignore in `.hound.yml`